### ISSUE TITLE
Dynamic Loads Functionality

### ISSUE THREAD

**Issue by LaggyHammer** [LaggyHammer/optical_char_recog#7]
Dynamic Loads functionality is mot accurate. Gives out extra outputs when fields are present between Dynamic Loads & Spring Constant

**Comment by amithtm** [comment]
https://github.com/tesseract-ocr/tesseract/wiki/ImproveQuality#page-segmentation-method

**Comment by LaggyHammer** [comment]
> https://github.com/tesseract-ocr/tesseract/wiki/ImproveQuality#page-segmentation-method

Yeah already tried those, 6 works the best (the one which is currently in use)

**Comment by amithtm** [comment]
https://www.danvk.org/2015/01/07/finding-blocks-of-text-in-an-image-using-python-opencv-and-numpy.html

We've to do something similar to this.
